<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HighlightTableVarDialog</class>
 <widget class="QDialog" name="HighlightTableVarDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>163</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Highlight Variable...</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Variable </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="varlist">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QPushButton" name="actionSelectColorMin">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="actionSelectColorMax">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Min:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Max:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
